kernels/microvm-kernel-x86_64-5.10.config

```diff
@@ -2156,7 +2156,7 @@ CONFIG_OVERLAY_FS_REDIRECT_ALWAYS_FOLLOW=y
 #
 # CD-ROM/DVD Filesystems
 #
-# CONFIG_ISO9660_FS is not set
+CONFIG_ISO9660_FS=y
```

Comment:
Can you confirm that this is the only change required in the config file of the kernel to work ?

Reply:
Yes, I recompiled the kernel using that config and everything works